package.json

```
@@ -144,11 +144,13 @@
 	"devDependencies": {
 		"@types/node": "^13.1.1",
 		"@types/vscode": "~1.20.0",
+		"markdownlint-cli": "0.20.0",
 		"typescript": "~3.5.2"
 	},
 	"scripts": {
 		"vscode:prepublish": "npm run compile",
 		"compile": "tsc -p ./",
+		"lint": "node_modules/.bin/markdownlint *.md",
```

Maybe you can simplify this to `markdownlint * .md`. NPM is smart enough to figure it out  😉

---

```suggestion
		"lint": "markdownlint * .md",
```

---

I tried that and it didn't work for me. That was the reason I changed it to `node_modules/.bin/markdownlint *.md`

---

Try `npx markdownlint *.md` instead.

---

That's weird... can you try it with `npx`
```suggestion
		"lint": "npx markdownlint *.md",
```

---

> That's weird

@dhonx I'm guessing you have `markdownlint` installed globally, whereas @Acid147 doesn't. Hence, it works for you but not for him. `npx` exists to solve this very problem.